Systems/Handbook/Tutorial/GuiHandbookTutorialPage.cs

```diff
@@ -149,16 +149,6 @@ public override void ComposePage(GuiComposer detailViewGui, ElementBounds textBo
             detailViewGui.AddRichtext(comps, textBounds, "richtext");
         }
 
-        public override float GetTextMatchWeight(string text)
-        {
-            /*string title = TextCacheTitle;
-            if (title.Equals(searchText, StringComparison.InvariantCultureIgnoreCase)) return 3;
-            if (title.StartsWith(searchText, StringComparison.InvariantCultureIgnoreCase)) return 2.5f;
-            if (title.CaseInsensitiveContains(searchText)) return 2;
-            if (TextCacheAll.CaseInsensitiveContains(searchText)) return 1;*/
-            return 0;
-        }
-

```

Comment:
I did not add the same override as I was not sure it's intended purpose, but obviously it's not difficult to bring back.

Reply:
`GetTextMatchWeight` is intended to return weighted results based on things like titles being higher in the list than descriptions.  Does your code adequately replace this weighting to make sure different things have different weights to them in the results?

Reply:
I understand the purpose of `GetTextMatchWeight`, I was specifically referring to the commented out code.

Yes, it sorts all pages by title matches first, then breaks ties by page content matches, and then by title length to prefer more exact matches.